Fix libretro with Vulkan

hrydgard opened the pull request:
This gets it going again, although not completely stable and it has some other minor issues.

However, it's a start.

Includes a new README_WINDOWS.txt file for getting started with building and debugging for libretro on Windows.

hrydgard committed:
Libretro/VK: Add a README for debugging, so I don't forget. Minor cleanups.

hrydgard committed:
Get Vulkan going again with libretro. Rendering is the wrong size and crashes on exit, but it's a start.

Only tested on Windows.

orbea commented:
I suppose I'll link this here for future reference.

https://github.com/hrydgard/ppsspp/issues/11836

aliaspider commented on Common/Vulkan/VulkanContext.cpp:
```diff
@@ -146,10 +146,14 @@ VkResult VulkanContext::CreateInstance(const CreateInfo &info) {
 	}
 
+	// Temporary hack for libretro. For some reason, when we try to load the functions from this extension,
+	// we get null pointers when running libretro. Quite strange.
```
`vkCreateInstance` is sort of stubbed with libretro since the frontend already created it, which means `instance_extensions_enabled_` is currently being ignored. 
I can't think of an easy way to solve this other than having the frontend always load the extension when available, the current libretro_vulkan api lacks the ability to allow the core to request specific instance extensions.

hrydgard replied:
Okay, that does make sense at least. We'll just have to deal with that then, there's no urgent need for that extension anyway.

hrydgard committed:
libretro: Fix screen size problem in Vulkan.

(Still something is slightly off, there are duplicated lines on screen.
Can't figure out what)

hrydgard commented:
I've fixed the screen size issue, making it a bit more playable.

There are some weird stability issues when unthrottling (space), and the sizing is not 100% accurate - there are some ugly duplicated lines. Not sure whose fault those are...

hrydgard commented:
Yeah run at 1x and play a game with detailed graphics, you'll notice that a couple of lines are missing or duplicated, it's like the vertical size is just a pixel or two off. 

But at least it's kinda usable now. I'm gonna merge.

aliaspider commented:
ah okay that is what you meant.

truns out `System_GetPropertyFloat` was defaulting to `-1` for unimplemented `SYSPROP*` which had some bad side effects.
#13548 should take care of it.
